docs/source/python/csv.rst

```diff
@@ -75,7 +75,16 @@ Customized conversion
 ---------------------
 
 To alter how CSV data is converted to Arrow types and data, you should create
-a :class:`ConvertOptions` instance and pass it to :func:`read_csv`.
+a :class:`ConvertOptions` instance and pass it to :func:`read_csv`::
+
+   table = csv.read_csv('testdata.csv', convert_options=pa.csv.ConvertOptions(
+    column_types={
+        's_name': pa.string(),
+        's_start_date': pa.string(),
+        'agg_percentile_bps': pa.decimal128(precision=38, scale=10),
+    }
+   ))
```

Comment:
It would be better if the example was related to the example above, IMHO.

Reply:
Specifically, you could maybe show to read the "total_bill" and "tip" columns as decimal?